gadgets/fdpass/README.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: fdpass
+sidebar_position: 0
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# fdpass
+
+The fdpass gadget traces file descriptor passing via a unix socket (`SCM_RIGHTS`).
```

Comment:
Could you please elaborate this a bit more? When a file descriptor passes via a unix socket, use cases when you want to trace this, issues it can identify, etc. Maybe also add some links to upstream documentation about `SCM_RIGHTS`.

Reply:
I updated the guide with two examples: D-Bus and runc.

Reply:
The bluetooth stack on Linux also use fd passing to pass a AF_BLUETOOTH socket to other processes. When I connect my bluetooth headset, the following happens:

```
COMM                    PID           SOCKET_INO     SOCKFD         FD FILE
bluetoothd             1231                28962          7         30 RFCOMM
dbus-broker            1227                40373         82         55 RFCOMM
pipewire               5353                43027         35         58 memfd:pipewire-memfd:flags=0x0000000f,type=2,size=2312
pipewire               5353                43027         35         59 [eventfd]
pipewire               5353                43027         35         60 [eventfd]
pipewire               5353                43027         35         61 [eventfd]
pipewire               5353                43027         35         63 [eventfd]
systemd-logind         1257              8826380         45         44 /dev/input/event21
systemd-logind         1257                29754         13         45 /dev/input/event21
dbus-broker            1227                37765         46         55 /dev/input/event21
```

It would be good to improve this gadget to show fds being received. With the previous output, we see that bluetoothd sends a AF_BLUETOOTH socket to dbus-broker, and then dbus-broker forward it to some other process, but we don't see which one. That could be added in a future PR.

Reply:
> I updated the guide with two examples: D-Bus and runc.
> 
Thank you! Do you think we can add a unit test doing the same?

Maybe a silly question, but I was expecting the FD to be the same number. Why it is not?

> That could be added in a future PR.
>
Once the PR is merged, could you please add an issue for it?

Reply:
> > I updated the guide with two examples: D-Bus and runc.
> 
> Thank you! Do you think we can add a unit test doing the same?

Yes, I can add a unit test.

> Maybe a silly question, but I was expecting the FD to be the same number. Why it is not?

What do you mean? Same number between the columns `SOCKFD` and `FD`? Or same number between different processes?

> > That could be added in a future PR.
> 
> Once the PR is merged, could you please add an issue for it?

Ok

Reply:
> forward it to some other process, but we don't see which one. That could be added in a future PR

Maybe add to the doc that this gadget currently only shows the "sender" of the `fd`. Showing the receiver would be great indeed